pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperSetupUtil.java

```diff
@@ -56,25 +86,61 @@ public class JuniperSetupUtil {
     private static String kitReturnId;
     private static String carrierId;
     private static String ddpKitRequestSettingsId;
-
     private static String instanceName;
+    private static String groupName;
     private static String studyGuid;
     private static String displayName;
     private static String collaboratorPrefix;
+    private static String userWithKitShippingAccess;
 
-    public JuniperSetupUtil(String instanceName, String studyGuid, String displayName, String collaboratorPrefix) {
+    public JuniperSetupUtil(String instanceName, String studyGuid, String displayName, String collaboratorPrefix, String groupName) {
         this.instanceName = instanceName;
         this.studyGuid = studyGuid;
         this.displayName = displayName;
         this.collaboratorPrefix = collaboratorPrefix;
+        this.groupName = groupName;
+    }
 
+    public void setupJuniperInstanceAndSettings() {
+        //everything should get inserted in one transaction
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult simpleResult = new SimpleResult();
+            try {
+                cmiAdminUtil.createRealmAndStudyGroup(instanceName, studyGuid, collaboratorPrefix, groupName);
+                ddpInstanceId = String.valueOf(cmiAdminUtil.getDdpInstanceId());
```

Comment:
There should be no reason to do this testing. It is the responsibility of UserAdminTestUtil to assure that it created these things or return an error.

Reply:
I don't undertsand... I need the ddpInstanceId to insert a bunch more stuff in other tables, does this makes sense as of why I am getting the ddpInstanceId from the User if that's the concern?

Reply:
That makes sense. It is unclear why you are testing the value for null.